packages/manager/src/features/IAM/Users/UserResources/UserResources.tsx

```diff
@@ -0,0 +1,46 @@
+import { Paper, Typography } from '@linode/ui';
+import Grid from '@mui/material/Grid';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { DocumentTitleSegment } from 'src/components/DocumentTitle';
+
+import { NO_ASSIGNED_RESOURCES_TEXT } from '../../Shared/constants';
+import { NoAssignedRoles } from '../../Shared/NoAssignedRoles/NoAssignedRoles';
+import { isObjNotEmpty } from '../../Shared/utilities';
+
+import type { IamUserPermissions } from '@linode/api-v4';
+
+interface Props {
+  assignedRoles: {} | IamUserPermissions;
+}
+
+export const UserResources = ({ assignedRoles }: Props) => {
+  const { username } = useParams<{ username: string }>();
+
+  const hasAssignedRoles = isObjNotEmpty(assignedRoles);
+
+  return (
+    <>
+      <DocumentTitleSegment segment={`${username} - User Resources`} />
+      <Paper>
+        <Grid
+          sx={{
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            marginBottom: '24px',
+          }}
+          container
+        >
+          <Typography variant="h2">Assigned Resources</Typography>
+        </Grid>
+
+        {hasAssignedRoles ? (
+          <p>UIE-8139 - RBAC-5: User Roles - Resources Table</p>
+        ) : (
+          <NoAssignedRoles text={NO_ASSIGNED_RESOURCES_TEXT} />
+        )}
```

Comment:
Similar feedback here about `Grid`. Let's try to use `Stack` for these simple layouts. 

```suggestion
        <Stack spacing={3}>
          <Typography variant="h2">Assigned Resources</Typography>
          {hasAssignedRoles ? (
            <p>UIE-8139 - RBAC-5: User Roles - Resources Table</p>
          ) : (
            <NoAssignedRoles text={NO_ASSIGNED_RESOURCES_TEXT} />
          )}
        </Stack>
```

I know this code is unfinished and likely to evolve, but I still think `Stack` is preferable here. For example, if we need to add a button, it will be as simple as:

```typescript
        <Stack spacing={3}>
          <Stack
            alignItems="center"
            direction="row"
            justifyContent="space-between"
          >
            <Typography variant="h2">Assigned Resources</Typography>
            <Button buttonType="primary">New Button</Button>
          </Stack>
          {hasAssignedRoles ? (
            <p>UIE-8139 - RBAC-5: User Roles - Resources Table</p>
          ) : (
            <NoAssignedRoles text={NO_ASSIGNED_RESOURCES_TEXT} />
          )}
        </Stack>
```

Reply:
sure, done